Gentoo bug: https://bugs.gentoo.org/show_bug.cgi?id=290024

Ephy does not display favicon of most websites due to a missing pattern match,
for "image/vnd.microsoft.icon" mimetype.

Index: epiphany/embed/ephy-favicon-cache.c
===================================================================
--- epiphany.orig/embed/ephy-favicon-cache.c	2009-05-21 09:53:41.207975209 -0400
+++ epiphany/embed/ephy-favicon-cache.c	    2009-05-21 09:54:28.061689298 -0400
@@ -696,6 +696,7 @@
 			return NULL;
 		}
 		valid = strcmp (mime_type, "image/x-ico") == 0 ||
+			strcmp (mime_type, "image/vnd.microsoft.icon") == 0 ||
 			strcmp (mime_type, "image/png") == 0 ||
 			strcmp (mime_type, "image/gif") == 0;
 		is_ao = strcmp (mime_type, "application/octet-stream") == 0;
